packages/teleport/src/DesktopSession/DesktopSession.tsx

```diff
@@ -162,6 +171,43 @@ export function DesktopSession(props: State) {
     );
   }
 
+  if (showAnotherSessionActiveDialog) {
+    return (
+      <Session {...props}>
+        <Dialog
+          dialogCss={() => ({ width: '484px' })}
+          onClose={() => {}}
+          open={true}
+        >
+          <DialogHeader style={{ flexDirection: 'column' }}>
+            Another Session Is Active
```

Comment:
```suggestion
            <DialogTitle>Another Session Is Active</DialogTitle>
```
it will give more emphasis, and make it consistent with our other dialogs